frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -206,10 +206,11 @@ class ChartSettings extends Component {
         },
       };
     } else if (currentWidget.props?.initialKey) {
-      const settings = this._getSettings();
-      const isBreakout = settings?.["graph.dimensions"]?.length > 1;
+      const hasBreakouts = series.some(
+        singleSeries => singleSeries.card?._breakoutColumn,
```

Comment:
`_breakoutColumn` is a part of the internal implementation of LineAreaBarCombo chart which patches the series. Can we avoid relying on this property? The RowChart does not set it, so it may break something

Reply:
@npfitz is it possible to add this phos `settings={this.props.visualizationSettings}` to this component instead?
https://github.com/metabase/metabase/blob/c6027c9eff47928acd97b6cc865cc09f745d4e93/frontend/src/metabase/query_builder/components/view/sidebars/ChartSettingsSidebar.jsx#L35-L48

This will make sure we propagate computed settings to `<ChartSettings />`

You could try this on master and I could no longer reproduce the issue using this repro:
- New Question -> Order
- Metric -> Count
- Group by -> 2 things. User.Source and Product.Category
- Visualize, and click settings
- Click the ellipsis by count, you'll see that it's offering you series settings for series[0]

Reply:
Interesting. I didn't know that we had access to those anywhere near `<ChartSettings>`. The safer approach may be to pass it in as another variable, there is 1 check on line 336 that, as far as I can tell, is assuming that settings may be empty. I'll just need to look into it a bit more

Reply:
I'm going to add in visualizationSettings as a separate prop called computedSettings. This is because we use the internal function `_getSettings` in a few places in `<ChartSettings/>`, notably when we compute to map when we update settings. Keeping these separate will ensure that we have a good distinction between stored and computed settings.

Reply:
That sounds good 👍